Remove/Modify tests for SVG transform with CSS syntax - part 2

dirkschulze opened the pull request.

dirkschulze committed:
Remove/Modify tests for SVG transform with CSS syntax - part 2

smfr commented on css/css-transforms/skewX/svg-skewx-020.html:
```diff
@@ -22,7 +22,7 @@
     <svg>
         <path d="M 2,1 100,1 198,99 101,99 Z" fill="red"/>
-        <rect width="100" height="100" fill="green" transform="skewX(0.0125e1turn)"/>
+        <rect width="100" height="100" fill="green"  style="transform: skewX(0.0125e1turn)"/>
```
Looks like you have two spaces in lots of places.

dirkschulze replied:
fixed

dirkschulze committed:
Add more tests for transform-origin

dirkschulze committed:
Remove additional space

dirkschulze commented:
Additional notes:
* Mostly removed units
* In some cases, removing units made no sense for the test anymore or the test got duplicated. Removed the test then.
* In cases where units were important or where transform functions were used not defined for the attribute, I switched to `style` attribute.
* Found an issue in Safari with transform and transform-origin attribute. Added tests with the properties instead. This shows that the feature actually is working but broken under certain conditions.

dirkschulze committed:
Fix translate tests

dirkschulze committed:
Revert incorrect changes, more fixes

dirkschulze committed:
Remove Transforms 2 tests from level 1 test suite

dirkschulze commented:
@smfr @grorg ping

smfr commented on css/css-transforms/animation/transform-interpolation-001.html:
```diff
@@ -1,7 +1,7 @@
 <meta charset="UTF-8">
 <title>transform interpolation</title>
-<link rel="help" href="https://drafts.csswg.org/css-transforms/#transform-property">
+<link rel="help" href="https://www.w3.org/TR/css-transforms-1/#transform-box#transform-property">
```
These links don't look right.

smfr replied:
These links look wrong,

dbaron commented:
@dirkschulze did you want to address the most recent review comment (and the merge conflicts)?
